thunder/torch/__init__.py

```diff
@@ -5206,7 +5206,7 @@ def mse_loss(
 
 # TODO Add annotations
 # NOTE The scale parameter is kwarg-only in PyTorch
-@torchsymbol(torch.nn.functional.scaled_dot_product_attention)
```

Comment:
I was wondering if just tagging `torch.nn.functional.sdpa` should be sufficient or is it necessary to tag the cudnn and sdpaex implementation as well.

I am not sure `recompute_saved_for_backward` is called before or after the transform_for_execution pass

Reply:
Maybe it would work to check the original bsym instead (but so to my mind, we might also have a transformation replacing the SDPA and then we need the tag on the thing that is written by the transform).

Reply:
Changed this to use the original bsym but left the tags on the implementation bsym, too.